CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # TigerBeetle Changelog
 
+## 2024-04-15 (No release: Queued up for upcoming multi-version binary release)
+
+### Safety And Performance
+
+- [#1810](https://github.com/tigerbeetle/tigerbeetle/pull/1810)
+
+  Incrementally recompute the number values to compact in the storage engine. This smooths out I/O
+  latency, giving a nice bump to transaction throughput under load.
+
+- [#1833](https://github.com/tigerbeetle/tigerbeetle/pull/1833)
+
+  Fix an infinite loop in the StorageChecker.
```

Comment:
"Fix" implies this was a bug... this was just removing a `while(true)` to follow tigerstyle.